src/components/layout/title/Layout.tsx

```
@@ -117,7 +106,20 @@ export default function TitleLayout() {
       break;
+    
+    // 모임 상세 페이지 - 동적 경로 처리
+    case /^\/meeting\/\d+\/?$/.test(pathname):
+      const meetingId = pathname.match(/\/meeting\/(\d+)/)?.[1];
+      headerProps = {
+        isBack: true,
+        isTitle: true,
+        titleText: meetingId || '모임 상세',
+        isLike: true,
+        isOut: true,
+      };
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)